### PR TITLE
Introduction to Operating System 

### LINKED ISSUE #2386: Introduction to Operating system in C/CPP domain
### Description

I will make a documentation on introduction of Operating System , its need .I will also explain some concepts like multitasking and multiprogramming.

Name-Megha Singh
Batch-1
 @Suvraneel Kindly assign me

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by am-dusky**
<hr>

## Description 📜
Topics covered:
1. What is operating system?
2. Need of operating system
3. Examples and types of operating system
4. Functions of operating system
5. Do we always need a manager
6. Multitasking and Multiprogramming

Fixes #2386

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP
<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by Suvraneel** [comment]
@kashishahuja2002 @gunjan-g

**Review by kashishahuja2002 — APPROVED** [review]
Combine both the files in one. 
Great content.

File: C_CPP/Operating System Concepts/Introduction_to_OS.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<li>Desktop- Windows, Linux, MacOS etc</li>
+<li>Mobile- Android , IOS etc</li>
+</ol>
+
+Types of Operating System :-
+<ol>
+<li>Batch Oprating System</li>
+<li>Time Sharing Oprating System</li>
+<li>Distributed Oprating System</li>
+<li>Network Oprating System</li>
```
Comment (Suvraneel) [inline C_CPP/Operating System Concepts/Introduction_to_OS.md:25]: Use 
- Item1
- Item2
`- Item3`

Use as less HTML as you can in md...

**Review by Suvraneel — CHANGES_REQUESTED** [review]

**Comment by Suvraneel** [comment]
@am-dusky Always prrofread you doc before submission

**Comment by am-dusky** [comment]
@Suvraneel @kashishahuja2002 I have corrected the mistakes.

**Comment by kashishahuja2002** [comment]
Ok, @Suvraneel

**Review by kashishahuja2002 — APPROVED** [review]

**Review by Suvraneel — CHANGES_REQUESTED** [review]
Update readme of OS directory

**Comment by am-dusky** [comment]
@Suvraneel thanku 🤩🎊💖

**Comment by kashishahuja2002** [comment]
Great work @am-dusky ❤️

**Pull request merged**